.github/workflows/main.yml

```
@@ -0,0 +1,57 @@
+name: Main
+
+on:
+  push:
+    branches:
+      - master
+      - spec_live
+  pull_request:
+    branches:
+      - master
```

I'd say you can simplify this a little - just use `pull_request:` plain.

```suggestion
on:
  pull_request:
  push:
    branches:
      - master
      - spec_live
```